src/phone-input/flag.js

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import React from 'react';
+
+import {SIZE} from './constants.js';
+import * as flagComponents from './flags/index.js';
+
+export default function Flag(props: {size: string, iso2: string}) {
+  const FlagComponent = flagComponents['Flag' + props.iso2.toUpperCase()];
+  const width = {
+    [SIZE.compact]: '22px',
+    [SIZE.default]: '30px',
+    [SIZE.large]: '37px',
+  }[props.size || SIZE.default];
```

Comment:
small nit, but would appreciate being more clear rather than immediately accessing the property after initialization. suggest moving the accessor to L21

Reply:
what were your thoughts behind in-lining string values rather than reading from theme? I worry about the flag icons being out of sync from alternate scales

Reply:
1. gotcha - i thought i saw the immediate accessor pattern in a few places throughout codebase so i used it here, but happy to move out as needed

2. i think i forgot to move those out - makes sense to have in theme for sure

Reply:
🤷‍♂ not a big deal